src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/process/DefaultProcessRunner.java

```diff
@@ -107,8 +110,10 @@ public void run(String[] command) throws ProcessRunnerException {
         processBuilder.environment().putAll(environment);
       }
 
+      processBuilder.directory(workingDirectory);
```

Comment:
That's a change to the JDK, since it's part of `java.lang.ProcessBuilder`. :smile: